include/sway/tree/view.h

```diff
@@ -64,10 +66,23 @@ enum sway_view_prop {
 	VIEW_PROP_INSTANCE,
 };
 
+struct sway_view_impl {
+	const char *(*get_prop)(struct sway_view *view,
+			enum sway_view_prop prop);
+	void (*set_size)(struct sway_view *view,
+			int width, int height);
+	void (*set_position)(struct sway_view *view,
+			double ox, double oy);
```

Comment:
I think the main advantage of this is that now the output layout can change and we don't have to adjust all of the coordinates of all the containers in the tree.

Reply:
Does it make sense to have a `double` for the view position?